universum/modules/error_state.py

```diff
@@ -21,10 +21,14 @@ class HasErrorState(Module):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)  # type: ignore
-        self.global_error_state = self.global_error_state_factory()
+        self.global_error_state: GlobalErrorState = self.global_error_state_factory()
 
     def error(self, message: str):
         self.global_error_state.errors.append(inspect.cleandoc(message))
 
-    def is_in_error_state(self):
+    def is_in_error_state(self) -> bool:
         return self.global_error_state.is_in_error_state()
+
+    def check_required_option(self, setting_name, message) -> None:
```

Comment:
why did you omit types in setting_name and message?

Reply:
Is there some mode for mypy to warn about this?

Reply:
--disallow-incomplete-defs